### DISCUSSION TITLE
Github Actions doesn't allow basic search in logs

### DISCUSSION THREAD

**Discussion by rmunn** [community/community#30766] [Actions]
This is a duplicate of https://github.com/orgs/community/discussions/3236, which is still going on (I experienced it just today). I opened this duplicate discussion to get https://github.com/orgs/community/discussions/3236 more attention, because it seems to have dropped off the GitHub team's radar. GitHub team, please close this once you've gotten https://github.com/orgs/community/discussions/3236 back on your radar. Thanks! Anyone else, please don't reply to this discussion, and reply to #3236 instead. Thanks!

**Comment by queenofcorgis** [comment]
Hi @rmunn, 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was locked because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
